### PR TITLE
Wilmer -> Feature/implement identityuser

### PR DESCRIPTION
**Pull request by Wil-JsDev**
## Implement EF Core Identity
- Add the library for `identity`.
- Each model class implement `identity`.
-  I changed the ApplicationDbContext for the dependency `identity`.
-  I removed class abstract User.
- I changed some type of data. 

## Database with Identity
image

### CODE REVIEW

File: backend/Infraestructure/IOC/IOCInfraestructure.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace Infraestructure.IOC
+{
+    public static class IOCInfraestructure
+    {
+        public static void AddPersistence(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddDbContext<ApplicationDbContext>(options =>
+            {
+                options.UseSqlite(configuration.GetConnectionString("SQLiteConection"), b => b.MigrationsAssembly("Infraestructure.IOC"));
```
Comment (ManasesLovera) [inline backend/Infraestructure/IOC/IOCInfraestructure.cs]: the connection string is: 'SQLiteConnection' you missed one 'n'.

**Review by ManasesLovera — CHANGES_REQUESTED** [review]
Please update as requested.

**Comment by Wil-JsDev** [comment]
Done!

**Comment by ManasesLovera** [comment]
All looks great now, good job!

**Pull request merged**